<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Lays</author>
 <class>BrPrint3D</class>
 <widget class="QMainWindow" name="BrPrint3D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>955</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>BrPrint3D</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Icons/Icons/logo</normaloff>:/Icons/Icons/logo</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="Portuguese" country="Brazil"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="BigButton" name="bt_open">
        <property name="text">
         <string>Open</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/Icons/Icons/openFile.png</normaloff>:/Icons/Icons/openFile.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="BigButton" name="bt_connect">
        <property name="text">
         <string>Connect</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/Icons/Icons/connect.png</normaloff>:/Icons/Icons/connect.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/Icons/Icons/brlogo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="BigButton" name="bt_play">
        <property name="text">
         <string>Play</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/Icons/Icons/play.png</normaloff>:/Icons/Icons/play.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="BigButton" name="bt_pause">
        <property name="text">
         <string>Pause</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/Icons/Icons/pause.png</normaloff>:/Icons/Icons/pause.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="BigButton" name="bt_stop">
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/Icons/Icons/stop.png</normaloff>:/Icons/Icons/stop.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="BigButton" name="bt_stopOnEmergency">
        <property name="text">
         <string>Emergency Stop</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/Icons/Icons/emergency.png</normaloff>:/Icons/Icons/emergency.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="bt_hide">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Settings - Show</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="PrinterSettingsWidget" name="_PrinterSettings" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
      <widget class="vtkWidget" name="vtkView" native="true"/>
      <widget class="ManualControlWidget" name="_ManualControl" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>&amp;Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionPortuguese"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionLegalWarning"/>
    <addaction name="actionDocumentation"/>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionEnglish">
   <property name="text">
    <string>&amp;English - EUA</string>
   </property>
  </action>
  <action name="actionPortuguese">
   <property name="text">
    <string>&amp;Portuguese - Brazil</string>
   </property>
  </action>
  <action name="actionLegalWarning">
   <property name="text">
    <string>&amp;Legal Warning</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>&amp;Documentation</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About Br-Print3D</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PrinterSettingsWidget</class>
   <extends>QWidget</extends>
   <header>PrinterSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ManualControlWidget</class>
   <extends>QWidget</extends>
   <header>ManualControlWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BigButton</class>
   <extends>QToolButton</extends>
   <header>BigButton.h</header>
  </customwidget>
  <customwidget>
   <class>vtkWidget</class>
   <extends>QWidget</extends>
   <header>vtkWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>bt_hide</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
